Patch 1 (#12)
project-error/projecterror.dev: docs/npwd/framework/qb-core.md

---

## Dependencies
Below are a list of dependencies, outside of the standard installation, for *NPWD* to work with a *QB-Core* server.
- [qb-npwd](https://github.com/qbcore-framework/qb-npwd)
    - Required for `npwd` to work with the `QB-Core` using the `newPlayer` export

>     - PhoneAsItem support

Reviewer: ```suggestion
    - Pre-configured phone as an item.
```

- [QB-Core](https://github.com/qbcore-framework/qb-core)

### Initial Setup

> The following steps are required for `qb-npwd` to function correctly with `npwd` and `qb-core`. Follow each step exactly as shown.

Reviewer: ```suggestion
The following steps are required for `qb-npwd` to function correctly with *NPWD* and `qb-core`. Follow each step exactly as shown.
```

> 1. Adjust the `config.json` file in `npwd` to match the example below *exactly*.

Reviewer: ```suggestion
:::warning 
Failure to adhere to these instructions will result in *NPWD* not working correctly.
 :::
1. Adjust the [config.json](https://github.com/project-error/npwd/blob/master/config.json) file in *NPWD* to match the example below.
```
